docs/Topics/mqtt.md

```diff
@@ -2,7 +2,7 @@
 
 # Message Queuing Telemetry Transport (MQTT) Eventing Support
 
-Open AMT Cloud Toolkit supports Eventing, customizable event monitoring, using Message Queuing Telemetry Transport (MQTT), an IoT publish-and-subscribe network protocol. With MQTT Eventing, administrators subscribe and customize network event monitoring. This eliminates the need to query or poll MPS to determine network events, such as a device's activation or deactivation. Administrators can subscribe to events and respond proactively. 
+Open AMT Cloud Toolkit supports Eventing, customizable event monitoring, using Message Queuing Telemetry Transport (MQTT), an IoT publish-and-subscribe network protocol. With MQTT Eventing, administrators subscribe and customize network event monitoring of server events. This eliminates the need to query or poll MPS to determine network events, such as a device's activation or deactivation. Administrators can subscribe to events and respond proactively. 
```

Comment:
I'm not sure what customize means here.  Users can't customize the events they receive.  They can subscript to specific topics (categories of events).

Reply:
So something like, "With MQTT Eventing, administrators can subscribe to specific topics (categories of events) for server event monitoring."?  I think the meaning of customize here == pick-and-choose which events you want.

Reply:
Yeah, I was thinking the customization was that they could make a decision what to subscribe to-- the categories. Is that wrong? The reason I use the word "customize" is that it generally speaks to having agency over something, and that tends to make people more satisfied than if they feel blocked by something.

Reply:
@bwendlandt-intel @matt-primrose I think the latest changes address your concerns.